fiftyone/core/utils.py

```diff
@@ -1562,7 +1567,7 @@ def get_default_batcher(iterable, progress=False, total=None):
             iterable,
             target_latency=target_latency,
             init_batch_size=1,
-            max_batch_beta=2.0,
+            max_batch_beta=8.0,
```

Comment:
@swheaton I bumped this to 8.0 to match the value you chose for the size-based batcher. Agree that it's a nice touch to get up to 1k batches in 4 << 10 steps

Reply:
ok sgtm just wanted to double check